lib/shoryuken/polling.rb

```diff
@@ -0,0 +1,88 @@
+module Shoryuken
+  module Polling
+    QueueConfiguration = Struct.new(:name, :options) do
+      def hash
+        name.hash
+      end
+
+      def ==(other)
```

Comment:
Implementing equality in this way meant that the tests did not have to change making the intent of the change more obvious.